src/ids-alert/ids-alert.scss

```
@@ -0,0 +1,47 @@
+@import '../../node_modules/ids-identity/dist/theme-uplift/tokens/web/theme-uplift-mixins.scss';
+@import '../../node_modules/ids-identity/dist/theme-uplift/tokens/web/theme-uplift.scss';
+@import '../ids-base/ids-base';
+
+:host {
+  @include mr-20();
```

```suggestion
  @include mr-8();
```

Would less padding work ok? Because 1. We want to work in 4s now (2, 4, 8, 16) and i think 20 seems like a lot and is something we would probably have to negate somehow when using the alerts later.